pgcrypto: tests fail with "encrypt error: Cipher cannot be initialized ?".

--- a/contrib/Makefile
+++ b/contrib/Makefile
@@ -35,7 +35,6 @@
 		pg_standby	\
 		pg_stat_statements \
 		pg_trgm		\
-		pgcrypto	\
 		pgrowlocks	\
 		pgstattuple	\
 		pg_visibility	\
